environments/demo/backend_lb_config.yaml

```diff
@@ -108,3 +108,7 @@ gateways:
         component: blob-router
       - product: reform-scan
         component: notification-service
+        
+      # IDAM Demo
+      - product: idam-api
```

Comment:
this would make url as idam-api-idam-api-demo.service.core-compute-demo.internal , i dont think you want that. Also, i thought u need ssl on idam api

Reply:
updated